src/e2e-tests/page-objects/smartthings/bedroomTab.page.js

```diff
@@ -0,0 +1,95 @@
+// bedroomTab.page.js
+import Page from '../page';
+import MenuComponent from '../menu.component';
+
+class BedroomTabPage extends Page {
```

Comment:
shall not extend page because it is not a page, it is more a component, set of options